modules/ducktests/tests/ignitetest/services/ignite.py

```diff
@@ -127,6 +141,36 @@ def clean_node(self, node):
         node.account.kill_java_processes(self.APP_SERVICE_CLASS, clean_shutdown=False, allow_fail=True)
         node.account.ssh("sudo rm -rf -- %s" % self.PERSISTENT_ROOT, allow_fail=False)
 
+    def rename_db(self, new_db_name: str):
+        """
+        Rename db.
+        """
+        for node in self.nodes:
+            self._rename_db(node, new_db_name)
+
+    def _rename_db(self, node, new_db_name: str):
+        """
+        Rename db.
+        """
+        assert len(self.pids(node)) == 0
+
+        node.account.ssh(f"mv {self.WORK_DIR}/db {self.WORK_DIR}/{new_db_name}")
+
+    def restore_from_snapshot(self, snapshot_name: str):
+        """
+        Copy from snapshot to db.
+        """
+        for node in self.nodes:
+            self._copy_snap_to_db(node, snapshot_name)
+
+    def _copy_snap_to_db(self, node, snapshot_name: str):
```

Comment:
snap?
why not cp_a_to_b?

why not inlined?

Reply:
fixed